tests/plugins/test_tvp.py

```diff
@@ -33,6 +32,11 @@ class TestPluginCanHandleUrlTVP(PluginCanHandleUrl):
             ("tvp_info", "https://www.tvp.info/78213165/euro-2024-polska-holandia-12-bartosz-salamon-czuje-sie-winny-za-te-porazke-wideo"),
             {},
         ),
+        # sport.tvp
+        (
+            ("tvp_sport", "https://sport.tvp.pl/79514191/paryz-2024-magda-linette-mirra-andriejewa-1-runda-na-zywo-transmisja-online-live-stream-igrzyska-olimpijskie-2872024"),
+            {},
+        )
```

Comment:
```suggestion
        # sport
        (
            ("tvp_sport", "https://sport.tvp.pl/79514191/paryz-2024-magda-linette-mirra-andriejewa-1-runda-na-zywo-transmisja-online-live-stream-igrzyska-olimpijskie-2872024"),
            {"stream_id": "79514191"},
        ),
```

- https://github.com/streamlink/streamlink/actions/runs/10132998279/job/28017929778?pr=6097#step:6:339
- https://github.com/streamlink/streamlink/actions/runs/10132998278/job/28017929129?pr=6097#step:5:33

Reply:
I run the pytest.